vktrace/src/vktrace_extensions/vktracevulkan/vkreplay/vkreplay_vkreplay.cpp

```diff
@@ -2463,6 +2463,7 @@ VkResult vkReplay::manually_replay_vkCreateXcbSurfaceKHR(packet_vkCreateXcbSurfa
     return replayResult;
 }
 
+#ifdef VK_USE_PLATFORM_XLIB_KHR
```

Comment:
Can't put an #ifdef around this function. It needs to be in the build of all platforms in order to support cross-platform playback. For example, vkreplay needs to support playing back on Linux a trace file created on Windows. The file vktrace_platform.h contains the type definitions needed for this cross-platform playback.

There are several more instances of this type of #ifdef in this pull request, I'm omitting comments on those, just marking them.